app/views/editions/show.html.erb

```diff
@@ -28,6 +28,14 @@
         <div role="tabpanel" class="tab-pane <% if active_tab.name == 'edit'%>active<% end %>" id="edit">
           <div class="link-check-report col-md-4">
             <%= render 'link_check_reports/link_check_report', edition: @edition, report: @edition.latest_link_check_report %>
+
+            <% if [AnswerEdition, GuideEdition, HelpPageEdition, LicenceEdition, LocalTransactionEdition, PlaceEdition, ProgrammeEdition, SimpleSmartAnswerEdition, TransactionEdition].include?(@edition.class) %>
```

Comment:
Do we know for sure if there are any types of edition which _don't include_ Govspeak fields? I've just had a quick look and can't seem to find any (but I could have missed something).

<details>
  <summary><strong>Helpful Terminal command</strong></summary>

> List models including `GOVSPEAK_FIELDS`:
> ```
> $ grep -rl GOVSPEAK_FIELDS app/models
> ```
> 
> And models without (use `-L` instead of `-l`):
> ```
> $ grep -rL GOVSPEAK_FIELDS app/models
> ```
</details>

I'm just thinking, this seems like a very long and specific looking list. And the code itself doesn't make it obvious why these ones are included and others aren't.

If there _are_ types of edition that we'd want to hide this for, I wonder if we could introspect the model to see if it has Govspeak fields. I think this could be a bit more intention revealing rather than relying on a hardcoded array.

```suggestion
            <% if @edition.class.const_defined?(:GOVSPEAK_FIELDS) %>
```

Reply:
yea good shout. Have added a constant to Editions